api/src/client/CigaretteLicenseEmailClient.ts

```diff
@@ -0,0 +1,104 @@
+import {
+  EmailConfirmationResponse,
+  EmailConfirmationSubmission,
+} from "@businessnjgovnavigator/shared";
+import { ApiPowerAutomateClientFactory } from "@client/ApiPowerAutomateClientFactory";
+import { EmailClient, HealthCheckMetadata } from "@domain/types";
+import { LogWriterType } from "@libs/logWriter";
+import { getConfigValue } from "@libs/ssmUtils";
+import { ReasonPhrases, StatusCodes } from "http-status-codes";
+
+const getConfig = async (): Promise<{
+  emailConfirmationUrl: string;
+  emailConfirmationKey: string;
+}> => {
+  return {
+    emailConfirmationUrl: await getConfigValue("cigarette_license_email_confirmation_url"),
+    emailConfirmationKey: await getConfigValue("cigarette_license_email_confirmation_key"),
+  };
+};
+
+export const CigaretteLicenseEmailClient = (logger: LogWriterType): EmailClient => {
+  const logId = logger.GetId();
+
+  const sendEmail = async (
+    postBody: EmailConfirmationSubmission,
+  ): Promise<EmailConfirmationResponse> => {
+    const config = await getConfig();
+
+    const cigarettePowerAutomateClient = ApiPowerAutomateClientFactory({
```

Comment:
Sand - I think this is a good canidate for dependency injection. The email client doesn’t need to know how the PowerAutomate client is built.